src/Interpreters/executeQuery.cpp

```diff
@@ -542,8 +279,11 @@ static std::tuple<ASTPtr, BlockIO> executeQueryImpl(
         }
 
         {
+            /// FIXME: does not supported by the QueryProcess
```

Comment:
What exactly is not supported?

Reply:
execute span for subqueries (i.e. it should be in QueryProcess but right now it is not)